Wikipedia/Code/OnThisDayCollectionViewCell.swift

```diff
@@ -1,16 +1,41 @@
 import UIKit
 
-@objc(WMFOnThisDayCollectionViewCell)
-class OnThisDayCollectionViewCell: SideScrollingCollectionViewCell {
-
-    let timelineView = OnThisDayTimelineView()
+@objc(WMFOnThisDayExploreCollectionViewCell)
+class OnThisDayExploreCollectionViewCell: OnThisDayCollectionViewCell {
 
-    @objc(configureForExploreWithOnThisDayEvent:previousEvent:dataStore:layoutOnly:)
-    func configureForExplore(with onThisDayEvent: WMFFeedOnThisDayEvent,  previousEvent: WMFFeedOnThisDayEvent, dataStore: MWKDataStore, layoutOnly: Bool) {
+    @objc(configureWithOnThisDayEvent:previousEvent:dataStore:layoutOnly:)
+    func configure(with onThisDayEvent: WMFFeedOnThisDayEvent,  previousEvent: WMFFeedOnThisDayEvent, dataStore: MWKDataStore, layoutOnly: Bool) {
         bottomTitleLabel.textColor = .wmf_blue
         bottomTitleLabel.text = previousEvent.yearWithEraString()
-        configure(with: onThisDayEvent, dataStore: dataStore, layoutOnly: layoutOnly, shouldAnimateDots: false)
+        super.configure(with: onThisDayEvent, dataStore: dataStore, layoutOnly: layoutOnly, shouldAnimateDots: false)
+    }
+
+    private lazy var whiteTopAndBottomGradientLayer: CAGradientLayer = {
+        let layer = CAGradientLayer.init()
```

Comment:
This can be written as `CAGradientLayer()` - also saw a few other places, for example `CGPoint.init(` that can be written as `CGPoint(`